deploy/service_account.yaml

```diff
@@ -2,3 +2,4 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: istio-operator
+  namespace: istio-operator
```

Comment:
this is redundant.  kubectl `-k` flag solves this with the kustomize.yaml code.

Reply:
Let's keep it since we will have an operator init command soon.

Reply:
I am not opposed to keeping the redundant and parts but I do want ppl to be able to deploy directly from the internet with just kubectl and no istioctl

Reply:
I am not opposed to keeping the redundant and parts but I do want ppl to be able to deploy directly from the internet with just kubectl and no istioctl

Reply:
ignore my comment.